src/Microsoft.AspNet.Authentication.OAuth/Notifications/OAuthAuthenticatedContext.cs

```diff
@@ -39,11 +36,6 @@ public class OAuthAuthenticatedContext : BaseContext<OAuthAuthenticationOptions>
         }
 
         /// <summary>
-        /// Gets the JSON-serialized user.
-        /// </summary>
-        public JObject User { get; protected set; }
```

Comment:
I agree.

If we really want to have a `User` property at the `OAuthAuthenticatedContext` level, then we should consider implementing a default user-info client in `CreateTicketAsync`, that would throw if it didn't receive a valid JSON payload, telling the user that he should implement his own notification delegate or his own handler.